src/controllers/CalculateFeeController.js

```diff
@@ -44,20 +21,8 @@ const isAccountHasExtraFee = async (address, node) => {
   }
 };
 
-const getConfig = async function () {
-  const response = await fetch(DEFAULT_FEE_CONFIG_URL);
-  return await response.json();
-};
-
-const getCachingFeeConfig = promiseCache(
-  CONFIG_EXPIRATION_TIME,
-  DEFAULT_FEE_CONFIG
-)(getConfig);
```

Comment:
Why don't we use remote config for fee values now?

Reply:
We decided at the meeting not to use remote config